torchao/quantization/prototype/qat.py

```diff
@@ -123,6 +155,7 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
             )
             return torch.nn.functional.linear(x_fq, w_fq)
     
+        # TODO: move this to common util
```

Comment:
right, this probably doesn't have to live here, I'm also adding a new util for this as well in quant_primitives.py as well